sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -145,20 +145,20 @@ def build_dataset(n_samples=50, n_features=200, n_informative_features=10,
     return X, y, X_test, y_test
 
 
-@pytest.mark.filterwarnings('ignore: The default value of cv')  # 0.22
 def test_lasso_cv():
     X, y, X_test, y_test = build_dataset()
     max_iter = 150
-    clf = LassoCV(n_alphas=10, eps=1e-3, max_iter=max_iter).fit(X, y)
+    clf = LassoCV(n_alphas=10, eps=1e-3, max_iter=max_iter, cv=3).fit(X, y)
```

Comment:
for curiosity, why change it here and not in all the other places where the default was used?

Reply:
The thing is that some tests have too few samples and it will fail with cv=5. I think this is easier to fix the cv then.